### ISSUE TITLE
Port old blog posts

### ISSUE THREAD

**Issue by matthewturk** [yt-project/blog#9]
Here is a listing of all the files in the old blog.  We can check these off as they go:

 - [x] 3DSurfacesAndSketchFab
 - [x] 2012InReview
 - [x] DevelopmentWorkshop2013
 - [x] ForkBlogging
 - [x] LineIntegralConvolution
 - [x] ParticleGenerator
 - [x] Relicensing
 - [x] Rockstar
 - [x] Simple_Grid_Refinement
 - [x] WhatsUpWith30
 - [x] governance
 - [x] objexporter
 - [x] post_00_volume-rendering-with-a-kd-tree-decomposition
 - [x] post_01_figuring-out-stereo-volume-rendering
 - [x] post_02_yt-has-moved-to-mercurial
 - [x] post_03_how-the-merger-tree-sped-up-sqlite-database-update-s
 - [x] post_04_quad-tree-projections
 - [x] post_05_improvements-to-parallelism
 - [x] post_06_kd-tree-rendering-improvements
 - [x] post_07_enzo-2-0-and-inline-yt
 - [x] post_08_amr-kd-tree-rendering-added-to-yt
 - [x] post_09_yt-development-star-particle-rendering-simple-merger-trees-and-documentation
 - [x] post_10_yt-development-2-0-cython-and-physics-module-wrapping
 - [x] post_11_yt-development-time-series-and-more
 - [x] post_12_yt-development-documentation
 - [x] post_13_yt-development-all-sky-column-density-calculation
 - [x] post_14_yt-development-healpix-and-contour-tree
 - [x] post_15_example-healpix-contour-rendering-movie
 - [x] post_16_yt-development-bitbucket-task-queues-and-streamlines
 - [x] post_17_yt-development-external-analysis-and-simulation-code-support
 - [x] post_18_yt-development-treecodes-guis-irc-and-more
 - [x] post_19_yt-2-1-release-announcement
 - [x] post_20_yt-logo-contest-and-possibly-a-free-mug
 - [x] post_21_yt-development-quad-trees-tickets-and-more
 - [x] post_22_gmaps-ify-your-data
 - [x] post_23_paraview-and-yt
 - [x] post_24_a-mission-statement-and-a-new-homepage
 - [x] post_25_a-movie-of-yt-in-paraview
 - [x] post_26_we-re-now-the-yt-project
 - [x] post_27_introducing-the-yt-hub
 - [x] post_28_this-blog-has-moved-but-you-don-t-have-to
 - [x] post_29_yt-2-2-release-announcement
 - [x] post_30_optimizing-yt
 - [x] post_31_dataset-tracking-with-yt
 - [x] post_32_new-screencasts
 - [x] post_33_announcing-the-2012-yt-workshop
 - [x] post_34_yt-testing-parallelism
 - [x] post_35_google-page-for-yt
 - [x] post_36_boolean-data-containers
 - [x] post_37_yt-workshop-registration-is-open
 - [x] post_38_yt-version-2-3-announcement
 - [x] post_39_workshop-in-just-a-week
 - [x] post_40_yt-workshop-2012-a-success
 - [x] post_41_what-s-new-with-yt
 - [x] post_42_yt-google-hangout-tomorrow
 - [x] post_43_yt-2-4-released
 - [x] yt2.5_released
 - [x] yt-dev-2013

**Comment by munkm** [comment]
Awesome!!! Thanks for making this so we can track our progress!!!

**Comment by matthewturk** [comment]
I think the hardest ones to convert will be the non-`post_` ones, since the `post` ones all have the metadata in them for time/author/etc.  The newer ones there aren't that many of.

One thing I'm annoyed by is that default pandoc doesn't convert images the way I want it to -- it uses some other method of inlining them.

**Comment by matthewturk** [comment]
`awk '{ sub(/[0-9]{10}/, strftime("%Y-%m-%dT%H:%M:%S-00:00", substr($0,10,10))) }1' file`

this will convert the dates, which I'm now doing ...

```
sed -i 's/.. author: \(.*\) <.*$/author: \1/' *.conv.rst
sed -i 's/.. date: /date: /' *.conv.rst
```

**Comment by matthewturk** [comment]
Applying this to the pre-processed old posts:

```
perl -0777p -e 's/^# (.*)\n\n^author: (.*)\n\n^date: (.*)\n/---\ntitle: \1\nauthor: \2\ndate: \3\nlastmod: \3\ncover: \/img\/random\/shoes.jpeg\ncategories:\n  - archive\ntags:\n  - archive\n---/mg'
```

**Comment by matthewturk** [comment]
So I've gotten the markdown converted for all of these, but I'm leaving them unchecked because we need to tag them and categorize them, and also probably come up with cover images.

**Comment by munkm** [comment]
resolved via munkm/yt-blog#11 -- I pushed a commit that updated the tags, author links, categories, and broken video URLs. We can go back and add short summaries later if we feel like it. We can also choose to remove either the archive category or tag, since one is redundant.
